_posts/blog/2018-04-13-conclusion-of-powers-of-tau.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "Conclusion of the Powers of Tau Ceremony"
+excerpt: "The world's largest multi-party computation ceremony comes to a successful end."
+categories: blog
+tags: []
+image:
+  feature: 
+date: 2018-04-13
+author: ebfull-jasondavies
+---
+
+In November of last year, we [announced](https://z.cash.foundation//blog/powers-of-tau/) the beginning of what has become the largest [multi-party computation](https://en.wikipedia.org/wiki/Secure_multi-party_computation) (MPC) ceremony ever performed. In the **Powers of Tau** ceremony, we aimed to produce partial public parameters that can be used by *all* projects that wish to use [zk-SNARKs](https://z.cash/technology/zksnarks.html) — small and flexible zero-knowledge proofs which require a parameter setup.
+
+We're happy to announce the completion of the ceremony, after many months and many dozens of participants. Today, we're publishing the final parameters, the protocol's transcript, a tool for verifying the transcript and the parameters, and a report of what took place.
+
+## Ceremony Results
+
+* We [announced](https://lists.z.cash.foundation/pipermail/zapps-wg/2018/000362.html) the completion of the ceremony on the mailing list, along with signatures of the ceremony artifacts.
+* The final parameters are available [over BitTorrent](https://s3-us-west-2.amazonaws.com/powersoftau-transcript/powersoftau.torrent).
+* The transcript of the ceremony is also available [over BitTorrent](https://s3-us-west-2.amazonaws.com/powersoftau-transcript/powersoftau-transcript.torrent). It has also been placed on the [Internet Archive](https://archive.org/details/transcript_201804). The transcript and the parameters can be verified using a [tool](https://github.com/ebfull/powersoftau/blob/master/src/bin/verify.rs) we have written.
+* The attestations of all participants are being collected [here](https://github.com/ZcashFoundation/powersoftau-attestations).
+
+### Ceremony Overview
+
+The ceremony used an MPC protocol described in a [paper](https://eprint.iacr.org/2017/1050) written by Ariel Gabizon, Ian Miers and Sean Bowe, who are scientists and engineers at the Zcash Company. This ceremony produced a partial common reference string (CRS) for Jens Groth's [pairing-based zk-SNARK scheme](https://eprint.iacr.org/2016/260) — the current state-of-the-art in performance. This CRS supports arithmetic circuits with up to 2^21 multiplication gates.
```

Comment:
Author ordering is wrong:Sean Bowe and Ariel Gabizon and Ian Miers

Reply:
Thanks @imichaelmiers, will fix in merge. :+1: